apps/portal/src/content/docs/components/toggle-group.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Toggle Group
+description: Toggle Group component documentation
+---
+
+The `ToggleGroup` component provides a set of toggle buttons that can be used to switch between different states or options. It is composed of subcomponents such as `ToggleGroup.Root` and `ToggleGroup.Item` to offer a structured and customizable interface.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`
+  <div className="flex flex-col space-y-8">
+    <section>
+      <div className="text-l font-semibold mb-4">Single Selection Example</div>
+      <ToggleGroup.Root type="single">
+        <ToggleGroup.Item value="1">Option 1</ToggleGroup.Item>
+        <ToggleGroup.Item value="2">Option 2</ToggleGroup.Item>
+        <ToggleGroup.Item value="3">Option 3</ToggleGroup.Item>
+      </ToggleGroup.Root>
+    </section>
+    <section>
+      <div className="text-l font-semibold mb-4">Multiple Selection Example with outline variant </div>
+      <ToggleGroup.Root type="multiple" variant="outline">
+        <ToggleGroup.Item value="1">Option 1</ToggleGroup.Item>
+        <ToggleGroup.Item value="2">Option 2</ToggleGroup.Item>
+        <ToggleGroup.Item value="3">Option 3</ToggleGroup.Item>
+      </ToggleGroup.Root>
+    </section>
+  </div>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { ToggleGroup } from '@harnessio/ui/components'
+
```

Comment:
Minor thing, but can we add `// ...` to show that this is not all that's required to implement the component?
```suggestion

// ...

```